puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/duffy.groovy

```diff
@@ -1,21 +1,39 @@
 def provision() {
     dir('foreman-infra') {
-        git url: 'https://github.com/ehelms/foreman-infra.git', branch: 'ci-centos-org'
+        git url: 'https://github.com/theforeman/foreman-infra.git'
     }
 
-	dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
+  	dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
         runPlaybook('provision.yml', 'localhost')
     }
 }
 
 def deprovision() {
-    if(fileExists('foreman-infra')) {
-        dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
+    if (fileExists('foreman-infra')) {
+      	dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
             runPlaybook('deprovision.yml', 'localhost')
         }
     }
 }
 
 def cico_inventory(relative_dir = '') {
-	return relative_dir + 'foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible/cico_inventory'
+  	return relative_dir + 'foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible/cico_inventory'
+}
+
+def ssh_config(relative_dir = '') {
+    return relative_dir + 'foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible/ssh_config'
+}
+
+def color_shell(command = '') {
+    wrap([$class: 'AnsiColorBuildWrapper', colorMapName: "xterm"]) {
+        sh "${command}"
+    }
+}
+
+def duffy_ssh(relative_dir = '', box_name, command = '') {
+    color_shell "ssh -F ${ssh_config(relative_dir)} ${box_name} '${command}'"
+}
+
+def duffy_scp(relative_dir = '', box_name, file_path, file_dest) {
```

Comment:
Similar tot above, the first parameters should always strive to be the required ones:

```
duffy_scp(file_path, file_dest, box_name, relative_dir='')
```